pip/py2-hepdata-lib.file

```diff
@@ -0,0 +1 @@
+%define PipPreBuild export CPPYY_BACKEND_LIBRARY=${ROOT_ROOT}/${PYTHON_LIB_SITE_PACKAGES}/libcppyy_backend.so
```

Comment:
@mrodozov , please set this in root.spec and drop these extra py[2,3]-[hepdata.rootpy]  files.